mabwiser/linear.py

```diff
@@ -72,32 +72,47 @@ def _scale_predict_context(self, x):
 
 class _LinTS(_RidgeRegression):
 
-    def predict(self, x):
+    def __init__(self, rng: np.random.RandomState, l2_lambda: Num = 1.0, alpha: Num = 1.0,
+                 scaler: Optional[Callable] = None):
+        super().__init__(rng, l2_lambda, alpha, scaler)
 
-        # Scale
-        if self.scaler is not None:
-            x = self._scale_predict_context(x)
+        # Set covariance to none
+        self.covar_decomposed = None
 
-        # Calculate the covariance matrix multiplied by alpha squred
+    def init(self, num_features):
+        super().init(num_features)
+
+        # Calculate exploration factor
         exploration = np.square(self.alpha) * self.A_inv
```

Comment:
I recommend putting lines 86-89 and 95-98 in a helper function to avoid code duplication

Reply:
That, and in case we use another decomposition (unlikely but..)

Reply:
Added _cholesky helper function as suggested.